moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/chomp_trajectory.h

```diff
@@ -129,6 +130,13 @@ class ChompTrajectory
    */
   void fillInCubicInterpolation();
 
+  /**
+  * \brief Fill the trajectory from a trajectory seed
```

Comment:
It'd be great if it described how the trajectory seed data needs to be packed in the request message.